### ISSUE TITLE
Accessibility improvements - Error and help messages

### ISSUE THREAD

**Issue by robinmolen** [open-formulieren/open-forms#4719]
### Thema / Theme

Frontend

### Omschrijving / Description

This issue is related to #4622

This issue targets the error and help message improvements of the accessibility rapport:

- [ ] 3.3.1
- [ ] 3.3.2
- [ ] 3.3.3

**Comment by robinmolen** [comment]
Regarding 3.3.3: The translation for the 'Invalid postcode' message has been changed, so all regular postcode fields will now show the expected postcode format. For the pre-fill postcode its a bit more complicated...

The pre-fill postcode doesn't use the validate config, it also isn't added to the DB. I expect its because the type is set to `'textfield'` (to allow the pre-fill functionality). So the error message, and validation, is pure because of the inputMask. I've added a custom error message for mask errors on this field, so new pre-fill postcode fields will have the correct error message.

We cannot change the default message for mask errors, because this then would cause issues for other fields with mask validation (i.e. bsn)
